apps/measures/models.py

```diff
@@ -518,9 +518,29 @@ class MeasuresDetailPage(Page):
     ]
 
     @property
-    def get_fields_of_action_display(self):
-        fields_of_action = [self.get_regenerative_management_display(),
-                            self.get_future_opportunities_for_all_display(),
-                            self.get_inclusive_shaping_of_urban_life_display(),
-                            self.get_facilitative_administration_display()]
-        return ', '.join(filter(None, fields_of_action))
+    def get_fields_of_action(self):
+        fields_of_action = self.get_selected_foa(
+            self.regenerative_management, 'reg',
+            REGENERATIVE_MANAGEMENT
+        ) + self.get_selected_foa(
+            self.future_opportunities_for_all, 'fut',
+            FUTURE_OPPORTUNITIES_FOR_ALL
+        ) + self.get_selected_foa(
+            self.inclusive_shaping_of_urban_life, 'inc',
+            INCLUSIVE_SHAPING_OF_URBAN_LIFE
+        ) + self.get_selected_foa(
+            self.facilitative_administration, 'fac',
+            FACILITATIVE_ADMINISTRATION
+        )
+        return fields_of_action
+
+    def get_selected_foa(self, selected_foa_ids, kind_id, kind):
```

Comment:
The `kind` did irritate me a bit. Can we rename that to maybe `set` or `category`? Or @philli-m has an idea how to name the groups or sets of fields of action?